app/Filament/Chantier/Resources/Chantiers/Pages/ViewChantier.php

```
@@ -14,182 +45,232 @@ class ViewChantier extends ViewRecord
                     $quote = $record->quote;
-                    if (!$quote) return;
+                    if (! $quote) {
+                        return;
+                    }
 
                     $percentage = $data['percentage'] / 100;
                     $amountHt = $quote->total_ht * $percentage;
                     $amountTva = $quote->total_tva * $percentage;
 
-                    $invoice = \App\Models\Commerce\CustomerInvoice::create([
+                    $invoice = CustomerInvoice::create([
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$invoice'. (undefined)

(UnusedLocalVariable)